test/test_pipeline/test_tabular_classification.py

```diff
@@ -442,3 +423,36 @@ def test_constant_pipeline_iris(fit_dictionary_tabular):
 
     assert val_score >= 0.8, run_summary.performance_tracker['val_metrics']
     assert train_score >= 0.8, run_summary.performance_tracker['train_metrics']
+
+
+@pytest.mark.parametrize("fit_dictionary_tabular_dummy", ['classification'], indirect=True)
+def test_pipeline_score(fit_dictionary_tabular_dummy):
+    """This test makes sure that the pipeline is able to achieve a decent score on dummy data
+    given the default configuration"""
+    X = fit_dictionary_tabular_dummy['X_train'].copy()
+    y = fit_dictionary_tabular_dummy['y_train'].copy()
+
+    # increase number of epochs to test for performance
+    fit_dictionary_tabular_dummy['epochs'] = 50
+
+    pipeline = TabularClassificationPipeline(
+        dataset_properties=fit_dictionary_tabular_dummy['dataset_properties'])
+
+    cs = pipeline.get_hyperparameter_search_space()
+    config = cs.get_default_configuration()
+    pipeline.set_hyperparameters(config)
+
+    pipeline.fit(fit_dictionary_tabular_dummy)
```

Comment:
I Think we removed the network check from then API, I meant [this](https://github.com/automl/Auto-PyTorch/blob/8447e917303ec64a180aa2c62e25ea1b3609a284/test/test_api/test_api.py#L137)

Can we add it here?

Reply:
yes I think it makes sense. Thanks